Framework/Beamline/src/DetectorInfo.cpp

```diff
@@ -61,12 +61,10 @@ bool DetectorInfo::isEquivalent(const DetectorInfo &other) const {
 
   // Scanning related fields. Not testing m_scanCounts and m_indexMap since
   // those just are internally derived from m_indices.
-  if (m_scanIntervals && other.m_scanIntervals &&
-      !(m_scanIntervals == other.m_scanIntervals) &&
-      (*m_scanIntervals != *other.m_scanIntervals))
+  if (m_scanIntervals != other.m_scanIntervals)
     return false;
-  if (m_indices && other.m_indices && !(m_indices == other.m_indices) &&
-      (*m_indices != *other.m_indices))
+  if (m_scanCounts && other.m_scanCounts &&
```

Comment:
Bug: `m_scanCounts` is not a pointer anymore so the `if` will not do what it used to do.